var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1050,7 +1050,8 @@ def include(self):
         return os.path.join("include", "python{}".format(self.version.up_to(2)))
 
     def setup_run_environment(self, env):
-        env.prepend_path("CPATH", os.pathsep.join(self.spec["python"].headers.directories))
+        # this call prevents us from using most python packages without dev headers
+        pass  # env.prepend_path("CPATH", os.pathsep.join(self.spec["python"].headers.directories))
```

Comment:
I would just delete the whole thing

Reply:
this was a mistake -- moving this to another PR